tests/Unit/Core/TypeTest.php

```diff
@@ -83,6 +83,42 @@ public function provideToString()
             'string[]',
             'array',
         ];
+
+        yield 'Nullable string' => [
+            Type::fromString('?string'),
+            '?string',
+            '?string',
+        ];
+
+        yield 'Nullable class' => [
+            Type::fromString('?Foobar'),
+            '?Foobar',
+            '?object',
+        ];
+
+        yield 'asd' => [
```

Comment:
asd? :)

Reply:
Don't know it's from yourself :)
Will fix it in `Nullable iterable class`

Reply:
Wonder what I was thinking ... :+1: